### PR TITLE
fix lead apidocs type

### PR DESCRIPTION
**Pull request by MeghanBomberger**
change highlight makes it look bigger than it is. Only the ( to [ on the get all and tabbing over changed

### CODE REVIEW

File: frontend/add-client/confirm.component.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ export default function Confirm(props: StepComponentProps) {
     employmentSector: c.employmentSector,
     payInterval: c.payInterval,
     weeklyEmployedHours: c.weeklyEmployedHours,
-    householdIncome: c.householdIncome,
+    householdIncome: c.householdIncome || null,
```
Comment (joeldenning) [inline frontend/add-client/confirm.component.tsx]: Why is this here?

Comment (MeghanBomberger) [reply frontend/add-client/confirm.component.tsx]: Does it not need to be to make it nullable?

Comment (joeldenning) [reply frontend/add-client/confirm.component.tsx]: I'm just confused why it's in this PR. The pr is for api docs, but this is inside of the frontend code.

Comment (MeghanBomberger) [reply frontend/add-client/confirm.component.tsx]: Shoot your right. I missed a git stash doing this one. That should only be in the other PR. Sorry about that.
